lms/templates/course_modes/track_selection.html

```diff
@@ -92,7 +92,11 @@ <h3 class="py-md-3">
 
                     <form class="form-register-choose mb-4 mt-0" method="post" name="enrollment_mode_form" id="enrollment_mode_form">
                         <div class="row">
-                        <div class="track-selection-options">
+                        % if is_verified_only or is_single_mode:
+                        <div class="grid-single">
+                        % else:
+                        <div class="grid-options">
+                        % endif
```

Comment:
I tried to add the class conditional at the HTML level in the <div> to write this in fewer lines but it the CSS selector would not get picked up after the DOM was loaded